WcaOnRails/config/routes.rb

```diff
@@ -42,6 +42,11 @@
     get 'register' => 'registrations#register'
     get 'register-require-sign-in' => 'registrations#register_require_sign_in'
   end
+
+  get 'competitions/:competition_id/report/edit' => 'delegate_reports#edit', as: :delegate_report_edit
+  get 'competitions/:competition_id/report' => 'delegate_reports#show', as: :delegate_report
+  patch 'competitions/:competition_id/report' => 'delegate_reports#update'
```

Comment:
I actually originally had this inside the resources block above. The problem is that I wanted their url helpers to be `delegate_report_path(competition)`, rather than `competition_delegate_report_path`. I think I was doing that while struggling in vain to get rid of the `url: ` parameter to the simple form in `WcaOnRails/app/views/delegate_reports/edit.html.erb`. Since the explicit url parameter is there anyways, I think we could move this stuff around again if we want to.

How exactly would you use `resources` here?